fineract-doc/build.gradle

```diff
@@ -44,14 +44,11 @@ asciidoctorj {
     version = '2.4.0'
 
     attributes = [
-        revnumber: {
-            project.version.toString()
-        },
         generated: "${buildDir}/generated/asciidoc",
         imagesdir: 'images',
         diagrams: 'diagrams',
-        years: '2015-2020',
-        revnumber: "$version".toString(),
+        years: '2015-2022',
+        revnumber: '1.7.0',
```

Comment:
hmmm... that way you always have to remember to set this version thing when you do a release... versioning is now handled by a Gradle plugin (in between releases you'll see Git hashes that clearly indicate against which commit this doc was created)... any specific reason why this should be fixed?